File: Bahasa C/PRAK204-2210817210013-Muhammad-Shandy-Dailami.c
#include <stdio.h>

int main(){
    float jariJariBejana, tinggiBejana, volume, phi, luas, keliling;
    printf("Masukkan jari-jari bejana: ");
    scanf("%f", &jariJariBejana);
    printf("Masukkan tinggi bejana: ");
    scanf("%f", &tinggiBejana);
    volume = 22 * jariJariBejana * jariJariBejana * tinggiBejana / 7;
    luas = 2 * 22 * jariJariBejana / 7 * (jariJariBejana + tinggiBejana);
    keliling = 2 * 22 * jariJariBejana / 7;
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}

File: Python/PRAK204-2210817210013-Muhammad-Shandy-Dailami.py
jariJariBejana = float(input("Masukkan jari-jari bejana: "))
tinggiBejana = float(input("Masukkan tinggi bejana: "))
phi = 22/7
volume = phi * jariJariBejana**2 * tinggiBejana 
luas = 2 * phi * jariJariBejana * (jariJariBejana + tinggiBejana)
keliling = 2 * phi * jariJariBejana
print("Volume = {:.2f}".format(volume))
print("Luas = {:.2f}".format(luas))
print("Keliling = {:.2f}".format(keliling))